docs/prisma.md

```diff
@@ -181,6 +181,13 @@ You can reset the database by running the following command:
 npx prisma migrate reset
 ```
 
+## Prototype Prisma schema
+```sh
+npx prisma db push
+```
+
+Also see https://www.prisma.io/docs/concepts/components/prisma-migrate/db-push
```

Comment:
When working on the schema, we found it was the easiest to modify the schema and push the schema if one wants to quickly test a change, because it removes the need to answer any interactive prompts in the console and doesn't create any temporary migration files.